src/ompl/util/RandomNumbers.h

```diff
@@ -152,6 +186,81 @@ namespace ompl
 
     };
 
+
+    /** \brief A class describing a prolate hyperspheroid, a special symmetric type of n-dimensional ellipse,
+    for use in direct informed sampling for problems seeking to minimize path length.
+    @par J D. Gammell, S. S. Srinivasa, T. D. Barfoot, "Informed RRT*: Optimal Sampling-based
+    Path Planning Focused via Direct Sampling of an Admissible Ellipsoidal Heuristic."
+    IROS 2014. DOI: <a href="http://dx.doi.org/10.1109/IROS.2014.6942976">10.1109/IROS.2014.6942976</a>.
+    <a href="http://www.youtube.com/watch?v=d7dX5MvDYTc">Illustration video</a>.
+    <a href="http://www.youtube.com/watch?v=nsl-5MZfwu4">Short description video</a>. */
+    class ProlateHyperspheroid
```

Comment:
This is purely a helper class for prolate hyperspheroid (PHS) sampling, but it seems like a lot to place in RandomNumbers.h, should I create a new file at  `src/ompl/util/ProlateHyperspheroid.h`? The reason I didn't do that, as this class seems sort of lost on its own...